pkg/storage/couchdb/errors.go

```diff
@@ -39,6 +39,7 @@ const (
 	// Error messages returned from Kivik CouchDB client that we directly check for
 	duplicateDBErrMsgFromKivik = "Precondition Failed: The database could not be created, the file already exists."
 	docNotFoundErrMsgFromKivik = "Not Found: missing"
+	docDeletedErrMsgFromKivik  = "Not Found: deleted"
```

Comment:
Do you have unit-test for this scenario "delete followed by query"?

Reply:
I'll add one right now